dev/devicelab/bin/tasks/ios_content_validation_test.dart

```diff
@@ -52,6 +50,13 @@ Future<void> main() async {
           fail('Failed to produce expected output at ${outputAppFrameworkBinary.path}');
         }
 
+        if (await bonjourServiceFound(outputAppPath)) {
+          throw TaskResult.failure('Release bundle has unexpected NSBonjourServices');
+        }
+        if (await localNetworkUsageFound(outputAppPath)) {
```

Comment:
Is this throwing an error if a user legitimately has their own mdns service for their release build?  We want to make sure to support that.

Reply:
This is an integration test, it doesn't have any mDNS service set.  The `xcode_build` script handles that case, it detects if the `NSBounjourServices` key is set, and adds it to the list.